src/Extensions/Microsoft.Diagnostics.Monitoring.Extension.Common/EgressHelper.cs

```diff
@@ -20,6 +21,9 @@ internal sealed class EgressHelper
     {
         private static Stream StdInStream;
         private static CancellationTokenSource CancelSource = new CancellationTokenSource();
+        private const int PayloadVersion = 1;
+
+        private const string GenericEgressFailureMessage = "The egress operation failed due to an internal error.";
```

Comment:
We currently don't have a Resx, and I wasn't sure what we want to expose to users for what's effectively an internal protocol error - I put this here temporarily, but we can do something else before this gets checked in.